repository/o.json

```diff
@@ -858,6 +858,23 @@
 				}
 			]
 		},
+		{
+			"name": "OpenSSL",
+			"details": "https://github.com/finitelife/openssl-sublimetext",
```

Comment:
The author and readme are just the defaults so we get those from the repo URL. So you can remove them here.